recipes/autometa/setup.py

```diff
@@ -0,0 +1,47 @@
+"""Setup for installation of Autometa."""
```

Comment:
If you haven't already done so, please make a PR to add this file to the original github repository.

Reply:
The original github repository dev branch contains the respective setup.py file from which the pre-release for this PR was constructed.

setup.py in original GH repo: https://github.com/KwanLab/Autometa/blob/dev/setup.py